.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+Closes # .
+
+## Summary of changes
+
+This added X awesome new feature....
+
+---
+### Tasks that must be compleeted:
+- [ ] Update changelog
```

Comment:
I rather leave updating the changelog to myself (or, at most, to us the maintainers).

Reply:
We should also mention the style guide.

Which reminds me, I've been also thinking about covering the commit messages in the style guide (or in a separate commit message guide): a summary of the general "good commit message" + the tagging approach I've landed on.  What do you think about this (and about my style of commit messages)?

Reply:
> I rather leave updating the changelog to myself (or, at most, to us the maintainers).

Ill take that out then. 

> Which reminds me, I've been also thinking about covering the commit messages in the style guide...

I haven't actually looked too closely at your commit messages beyond noticing that they are frequently multiple lines. 
I generally just do frequent `git commit -am "my amazing changes here"` and keep them fairly short and I think they are human readable but not all that descriptive .